### DISCUSSION TITLE
Apt packages for arm64 / rpi 5 / 4

### DISCUSSION THREAD

**Discussion by Kitki30** [Kong/insomnia#8066] [Ideas] [answered]
Can you compile apt packages for rpi 5 / arm64.
My computer did broke (cooler pump has broken), and i need to use rpi 5.
I did download amd64 but it doesnt install in flatpaks.

**Comment by jackkav** [comment, marked as answer]
This is done https://github.com/Kong/insomnia/releases/download/core%4010.3.1/inso-linux-arm64-10.3.1.tar.xz

**Reply by Kitki30** [reply]
thanks!
